docs/man.rst

```diff
@@ -26,178 +26,154 @@ To install the Quipucords server and command line interfaced (CLI) client:
-
-``-e server_name=quipucords2``
 
-  Sets the name for the Quipucords server. Defaults to ``quipucords``.
-
-``-e server_http_timeout=90``
-
-  Sets the HTTP timeout length for the Quipucords server. Defaults to ``120``.
-
-``-e inspect_job_timeout=10700``
-
-  Sets the network inspection scan timeout in seconds. Defaults to ``10800`` (3 hours).
-
-``-e connect_job_timeout=500``
-
-  Sets the network connection scan timeout in seconds. Defaults to ``600`` (10 minutes).
-
-``-e ansible_log_level=10``
-
-  Sets the level of log output by Ansible during network scans. Defaults to ``0``, which is the value for no extended logs.
-
-Installing offline
-------------------
+Installing the server offline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 If you choose the offline option to run the install command, you must do the following steps:
 
 #. Obtain the installation packages on a machine with internet connectivity.
+  - Download the following server image: https://github.com/quipucords/quipucords/releases/latest/download/quipucords_server_image.tar.gz
+  - Create the PostgreSQL image TAR file with Docker. Use the the following command, where the package name is ``postgres.9.6.10.tar``::
 
+      docker pull postgres:9.6.10 && docker save -o postgres.9.6.10.tar postgres:9.6.10
 #. Create a location for the packages on the machine where Quipucords will be installed and move the packages to that location.
+#. Run the qpc-tools with the required options to complete an offline installation.  For example::
+
+    qpc-tools server install --offline --offline-files='/PATH_TO_OFFLINE_FILES' --version=0.9.1
 
-#. Run the qpc-tools with the required options to complete an offline installation.
 
-Obtaining packages
-~~~~~~~~~~~~~~~~~~
-Download the following packages to the machine with internet connectivity. Make sure that the package names match the default names in the following instructions.
+Installing a specific version of the server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+By default, the ``qpc-tools server install`` command installs the latest release unless an earlier version is specified in the command. For example, if the previous version of Quipucords that you want to install is 0.9.0., you would enter the following command::
 
-*Quipucords server*
+    qpc-tools server install --version=0.9.0
 
-#. Go to the following URL: https://github.com/quipucords/quipucords/releases
+Command Line Interface (CLI)
+----------------------------
 
-#. Download the ``quipucords_server_image.tar.gz`` package.
+CLI Installation
+^^^^^^^^^^^^^^^^
+The ``qpc-tools cli install`` command with no options performs a basic installation with the preset defaults.
 
-*PostgreSQL*
+Note that in log information for the ``qpc-tools cli install`` command, references to ``quipucords server`` are relevant to the Quipucords server, and references to ``qpc CLI`` are relevant to the Quipucords command line interface client.
```

Comment:
Should we be lowercasing qpc here? Maybe change `Note that in log information` to `Note that in the log information`

Reply:
ack